### DISCUSSION TITLE
Just downloaded copilot and signed up for it but experiencing issues

### DISCUSSION THREAD

**Discussion by bray4403** [community/community#35211] [Copilot Conversations]
<img width="461" alt="copilot_issue" src="https://user-images.githubusercontent.com/114193851/194105066-81cfa397-1cb1-4d4d-b4cc-37a5db00ee25.png">
I started a free trial and authorized copilot on my vs code and linked my accounts but this message is popping up.

**Comment by iamcjwilliams** [comment]
@bray4403 Can you provide a screenshot of the output log?

**Comment by bray4403** [comment]
Got it resolved, thank you though.
